docs/1-get-started.md

```diff
@@ -0,0 +1,35 @@
+# Getting started
+
+## Installation
+
+This library is built atop of [PSR-7](https://www.php-fig.org/psr/psr-7/) and
+[PSR-18](https://www.php-fig.org/psr/psr-18/). So you will need to install some
+implementations for those standard interfaces.
+
+If no PSR-18 client or PSR-7 message factory is available yet in your project
+or you don't know or don't care which one to use, just install some default:
+
+```bash
+composer require symfony/http-client nyholm/psr7
+```
+
+You can now install the Slack client:
+
+```bash
+composer require jolicode/slack-php-api
+```
+
+## Quick start
+
+```php
```

Comment:
Before that snippet, a link to the "get your token" is needed. There is no "quick start" without token IMO.

Reply:
Okay, I will move the token chapter here :+1: